cmd/update.go

```diff
@@ -32,6 +33,7 @@ under $GOPATH/bin and automatically updates commands to the latest version.`,
 	}
 	cmd.Flags().BoolP("dry-run", "n", false, "perform the trial update with no changes")
 	cmd.Flags().BoolP("notify", "N", false, "enable desktop notifications")
+	cmd.Flags().StringP("exclude", "e", "", "specify binaries which should not be updated separated using ',' without spaces as a delimiter")
```

Comment:
Wouldn't it be more convenient to use StringSliceP() instead of StringP() for this code?
https://pkg.go.dev/github.com/spf13/pflag#StringSliceP

Reply:
I might be better, I just never used cobra and didn't take the time to read through the docu.